week2/todoList/todo.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TodoList</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+</head>
+<body>
+    <header class="logo">
+        <h2>Todo List</h2>
+    </header>
+    <nav>
+        <button class="btn-today">오늘만 보기</button>
+        <button class="btn-tomorrow">내일만 보기</button>
+        <button class="btn-all">함께 보기</button>
+    </nav>
+    <main>
+        <section class="left">
```

Comment:
```suggestion
        <section class="main-section left-section">
```
이렇게 메인 섹션임을 나타내는 클래스, 좌우 섹션을 구분해주는 2개의 클래스를 지정해서  아래 onAdd함수에서 closest메서드를 통해 해당요소를 참조할 수 있을것같아요!!